checknoglobals/check_no_globals.go

```diff
@@ -86,16 +87,23 @@ func isAllowedSelectorExpression(v *ast.SelectorExpr) bool {
 	return false
 }
 
-// looksLikeError returns true if the AST identifier starts
-// with 'err' or 'Err', or false otherwise.
-//
-// TODO: https://github.com/leighmcculloch/gochecknoglobals/issues/5
-func looksLikeError(i *ast.Ident) bool {
+// isError reports whether the AST identifier
+// starts with 'err' or 'Err' and implements the error interface.
+func isError(i *ast.Ident, ti *types.Info) bool {
 	prefix := "err"
 	if i.IsExported() {
 		prefix = "Err"
 	}
-	return strings.HasPrefix(i.Name, prefix)
+
+	hasErrPrefix := strings.HasPrefix(i.Name, prefix)
+	if !hasErrPrefix {
+		return false
+	}
```

Comment:
@leonklingele @bombsimon Do you folks have thoughts on if we should remove the err/Err prefix requirement? Regardless of the name given to the global variable, just checking if it is an error seems reasonable to me.

Reply:
I'd leave it as-is. We could make it configurable with a default that still enforces the prefix but I don't really see a benefit in doing so.